README.md

```diff
@@ -17,6 +17,7 @@ To add a conference to the list, please [contribute](#contributing)!
 | [Data Governance Summit 2018](http://datagovernance.in/index.html) | 8 Feb | JW Marriott, Mumbai | The conference which give insights of the current challenges being faced regarding data quality and security. |
 | [Hosting and Domain Conference India](http://www.hdcon.org/) | 3 Feb | Hyderabad | Hosting and Domain Conference is the World's Top initiative to discuss on hosting, domain, blogging and web technologies. |
 | [ReactFoo Pune](https://reactfoo.in/) | 19-20 Jan | Pune | A conference on the lessons learned while implementing React and React Native. |
+| [GHC 18](https://ghc.anitab.org/) | 26–28 Sep | Houston, TX | The Grace Hopper Celebration is the world's largest gathering of women technologists. |
```

Comment:
Thanks for adding this, @tanya-vedi! However @kspviswa is right, this is only for conferences happening in India.

And from what I see for GHCI (https://ghcindia.anitab.org/), the 2018 version has not been announced yet.

Closing this PR. Feel free to circle back to add when GHCI is announced. Thanks!